internal/cli/terraform.go

```diff
@@ -82,17 +89,38 @@ func generateTerraformCmdRun(cli *cli, inputs *terraformInputs) func(cmd *cobra.
 			return err
 		}
 
-		if err := generateTerraformConfigFiles(inputs.OutputDIR, data); err != nil {
+		if err := generateTerraformImportConfig(inputs.OutputDIR, data); err != nil {
 			return err
 		}
 
-		cli.renderer.Infof("Terraform config files generated successfully.")
+		if terraformProviderCredentialsAreAvailable() {
+			if err := generateTerraformResourceConfig(cmd.Context(), inputs.OutputDIR); err != nil {
+				return err
+			}
+
+			cli.renderer.Infof("Terraform resource config files generated successfully in: %q", inputs.OutputDIR)
+			cli.renderer.Infof(
+				"Review the config and generate the terraform state by running: \n\n	cd %s && ./terraform apply",
```

Comment:
Might be nice to avoid the `cd` instruction if currently in the target directory.

Reply:
Unnecessary extra logic IMO, doing a cd . won't fail.